Firestore/core/src/firebase/firestore/local/local_documents_view.h

```diff
@@ -78,7 +80,7 @@ class LocalDocumentsView {
    * @param since_read_time If not set to SnapshotVersion::None(), return only
    *     documents that have been read since this snapshot version (exclusive).
    */
-  model::DocumentMap GetDocumentsMatchingQuery(
+  virtual model::DocumentMap GetDocumentsMatchingQuery(
```

Comment:
Done (added "// Virtual for testing"). The only reason I didn't add it right away is that I don't like having both "/** */" and "// " style comments, while also not being a huge fan of polluting the API doc with this information.

Reply:
I understand your concerns, but since in C++ member functions aren't virtual by default, I think it's worthwhile to call out whether it's intended for overriding in the general case.